junit-platform-engine/src/main/java/org/junit/platform/engine/support/hierarchical/NodeTestTask.java

```diff
@@ -165,13 +165,25 @@ private void cleanUp() {
 
 	private void reportCompletion() {
 		if (throwableCollector.isEmpty() && skipResult.isSkipped()) {
+			try {
+				node.nodeSkipped(context, testDescriptor, skipResult);
+			}
+			finally {
+				//swallow
+			}
 			listener.executionSkipped(testDescriptor, skipResult.getReason().orElse("<unknown>"));
 			return;
 		}
 		if (!started) {
 			// Call executionStarted first to comply with the contract of EngineExecutionListener.
 			listener.executionStarted(testDescriptor);
 		}
+		try {
```

Comment:
What is this try / finally block good for, it doesn't swallow the exception like the comment indicates.
Shouldn't this be a try / catch(Exception e) block to really swallow the exception?

Reply:
Fixed it, typo from some resource handling habits. Thanks!